cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSink.java

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
CNR entry about the changes in this PR should be added.

Reply:
Fixed in [d405b06](https://github.com/telefonicaid/fiware-cygnus/pull/1799/commits/d405b0669fc29cd264e7e161236606bc3b8b75b9)